code/Model/SiteTree.php

```diff
@@ -136,6 +136,7 @@ class SiteTree extends DataObject implements PermissionProvider, i18nEntityProvi
      * Used as a cache for `SiteTree::allowedChildren()`
      * Drastically reduces admin page load when there are a lot of page types
      * @var array
+     * @deprecated 5.4.0 will be moved to SilverStripe\ORM\Hierarchy\Hierarchy->cache_allowedChildren
      */
     protected static $_allowedChildren = [];
```

Comment:
We don't really have a process for deprecating properties - and this one is very unlikely to be used by downstream devs anyway. I won't mention it in the changelog unless asked to.